src/control/server/instance_test.go

```diff
@@ -15,14 +16,25 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/pkg/errors"
+	"google.golang.org/protobuf/proto"
 
 	"github.com/daos-stack/daos/src/control/common"
+	commonpb "github.com/daos-stack/daos/src/control/common/proto"
+	ctlpb "github.com/daos-stack/daos/src/control/common/proto/ctl"
 	srvpb "github.com/daos-stack/daos/src/control/common/proto/srv"
+	"github.com/daos-stack/daos/src/control/drpc"
+	"github.com/daos-stack/daos/src/control/lib/atm"
 	"github.com/daos-stack/daos/src/control/logging"
 	"github.com/daos-stack/daos/src/control/server/engine"
+	"github.com/daos-stack/daos/src/control/server/storage"
+	"github.com/daos-stack/daos/src/control/server/storage/bdev"
 	"github.com/daos-stack/daos/src/control/system"
 )
 
+var (
+	_ Engine = (*MockInstance)(nil)
+)
+
```

Comment:
Took me a few minutes to realize this was probably for catching at compile time if the mock doesn't get updated. Mind adding a comment?

Reply:
Done.